go.mod

```
@@ -3,44 +3,44 @@ module k8c.io/kubeone
 go 1.18
```

```suggestion
go 1.19
```

I know it's somewhat unrelated but would be great if we can update it in this PR as well.